diff --git a/common/randombytes.c b/common/randombytes.c
index ba60a12..6cc694b 100644
--- a/common/randombytes.c
+++ b/common/randombytes.c
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0 or CC0-1.0
 #include "randombytes.h"
 
-#if defined(STM32F2) || defined(STM32F4) || defined(STM32L4R5ZI) && !defined(MPS2_AN386)
+#if defined(STM32F2) || defined(STM32F4) || defined(STM32F7) || defined(STM32L4R5ZI) && !defined(MPS2_AN386)
 
 #include <libopencm3/stm32/rng.h>
 
